### PR TITLE
esm v10

### PR DESCRIPTION
**Pull request by asafshen**
In progress

### CODE REVIEW

File: Integrations/integration-PostgreSQL.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ script:
         conn = pgsqlConnect()
         try:
             cur = conn.cursor()
-            cur.execute(demisto.args()['query'])
+            cur.execute(demisto.args()['query'])Å
```
Comment (aviadl) [inline Integrations/integration-PostgreSQL.yml]: @asafshen you have here a typo, please fix it

File: Integrations/integration-ESM-v10.yml [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+            cmd = 'alarmGetTriggeredAlarmsPaged?pageSize=500&pageNumber=1&triggeredTimeRange=' + time_range
+            user = self.get_user()
+
+            query = json.dumps({ 'assignedUser' : user})
+
+            res = self.cmdquery(cmd, query)
+
+            '''
+            # mock of the result
```
Comment (meirwah) [inline Integrations/integration-ESM-v10.yml]: emmm.. not sure you want to leave this here...

File: Integrations/integration-ESM-v10.yml [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+        sys.exit(0)
+    if demisto.command() == 'esm-fetch-fields':
+        res = esm.featchAllFields()
+        demisto.results(res)
+        sys.exit(0)
+    if demisto.command() == 'esm-search':
+        args = demisto.args()
+        res = esm.search(
+            demisto.get(args, 'timeRange'),
+            json.loads(demisto.get(args, 'filters')),
```
Comment (meirwah) [inline Integrations/integration-ESM-v10.yml]: if filters arg is empty, you get empty json here?

Comment (asafshen) [inline Integrations/integration-ESM-v10.yml]: we get an error if filter is empty

**Comment by Spivaka** [comment]
Remove go module in the server as well

**Comment by meirwah** [comment]
Wait, the Go impl supports version 9.X, while this integration supports version 10.x only.
So don't remove until you can support both versions in BYOI (or keep things as are)

**Comment by asafshen** [comment]
@meirwah @Spivaka 
Please review 
also updated the description of nitro integration (support up to 9.X)

File: Integrations/integration-ESM-v10.yml [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+description: Run queries and receive alarms from Intel Security ESM
```
Comment (meirwah) [inline Integrations/integration-ESM-v10.yml]: add here, for ESM version 10.x ONLY

File: Integrations/integration-ESM-v10.yml [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+        sys.exit(0)
+    if demisto.command() == 'fetch-incidents':
+
+        last_run = demisto.getLastRun()
+        start = last_run and last_run['value']
+
+        if (not start):
+            start = (datetime.datetime.now() - datetime.timedelta(minutes=1)).isoformat()
+
+        end = datetime.datetime.now().isoformat()
```
Comment (meirwah) [inline Integrations/integration-ESM-v10.yml]: if ESM if in a different time zone then the demisto server, you will have an issue here.
you should add a config param to take time -/+ the time zone delta

Comment (asafshen) [inline Integrations/integration-ESM-v10.yml]: @meirwah will check that against our instance
I noticed that in go implementation we use zulu time..

why shouldn't we do this here also?

**Pull request merged**